drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -225,51 +227,96 @@ Error AudioDriverPulseAudio::init_device() {
 	return OK;
 }
 
+#define LIBPULSE_DLSYM(name)                                                 \
+	libpulse_##name = (name##_t)dlsym(libpulse, #name);                      \
+	if (!libpulse_##name) {                                                  \
+		fprintf(stderr, "could not find symbol %s: %s\n", #name, dlerror()); \
+		return FAILED;                                                       \
+	}
+
 Error AudioDriverPulseAudio::init() {
 
+	libpulse = dlopen("libpulse.so", RTLD_LAZY);
```

Comment:
That won't work if people don't have the development library installed (`libpulse-dev` or similar) which is the one containing the `.so` symlink. You'll have to hardcode `libpulse.so.0` AFAIK.